### PR TITLE
created business value summary page

### PR DESCRIPTION
**Pull request by edwardearle**

### CODE REVIEW

File: docs/Ways-of-Working/Learning/Business-Value.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+reviewed: 
+reviewer:
+next-review: 2024-12-01
+---
+
+Every organisation produces "Business Value". The term refers to anything that supports the health of the organisation in the long term. For a commercial enterprise this usually means sustained revenue growth and profitability, although it may also be tightly linked to the organisation's mission.
+
+Ideally, all the work done by and for an organisation should enable (directly or indirectly) the creation or maintenance of business value. Work that does not is either wasteful or damaging.
+
+However! It may not always be apparent how work supports or negates the creation of value, especially where it isn't clear what business value actually it.
```
Comment (jamesnwarner) [inline docs/Ways-of-Working/Learning/Business-Value.md:14]: Small typo - should be "especially where it isn't clear what business value actually **is.**"

File: docs/Ways-of-Working/Learning/Business-Value.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+???+ example
+    A team perform an experiment which turns out not to create the expected positive outcome.
+    
+    This work could be perceived as wasteful or even damaging. However, the team have gained valuable knowledge that could improve their ability to support the health of the organisation in the long term (business value generated). If the team fail to leverage this knowledge, or even carry on with their experiment regardless of its outcome, it may well in fact be wasteful or damaging (business value lost).
+
+Creating a link between Business Value and the work we do helps to guide us to do more valuable work, and even improve our ability to do more valuable work. Not only is does this make our organisation more effective, but it can improve individual motivation, job satisfaction, and happiness.
```
Comment (jamesnwarner) [inline docs/Ways-of-Working/Learning/Business-Value.md:21]: Extra word "is" in the sentence "Not only **is** does this make our organisation..."

File: docs/Ways-of-Working/Learning/Business-Value.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## The Topology of Business Value
+
+This model aims to demonstrate the link between business value and the common structures used in software delivery environments. Visualising this model can help us to express this link and its importance, as well as help us to explore how a specific things are relate.
```
Comment (jamesnwarner) [inline docs/Ways-of-Working/Learning/Business-Value.md:25]: Minor typo - should read "as well as help us to explore **how specific things are related**.

**Review by edwardearle — COMMENTED** [review]
Added change suggestions based on JW review

**Pull request merged**